app/assets/javascripts/staff/events.js

```diff
@@ -7,5 +7,9 @@ $(document).ready(function () {
   $('.cancel-status-change').click(function() {
     $('.status-dropdown').hide();
     $('.btn-nav').show();
-  })
-});
\ No newline at end of file
+  });
+
+  $('.modal').on('keypress', function (e) {
+    if (e.which == 13) $(this).submit();
+  });
```

Comment:
Ah yep - the invite one isn't working, you're right. I thought I checked all of them, but I guess I missed this one. I'll check into it.